docs/pages/access-controls/sso.mdx

```diff
@@ -249,6 +249,17 @@ or </ScopedBlock>create a `cluster_auth_preference` resource.
       # Set as saml, oidc, or github
       type: saml|oidc|github
   ```
+
+  <Admonition type="note" title="IdP-initiated SSO">
+    Enabling the `allow_idp_initiated` flag in SAML connectors allows users to
+    login to Teleport with one click from the dashboard provided by the IdP.
+
+    This feature is potentially unsafe and should be used with caution.
+
+    Enabling IdP-initiated login comes with notable security risks such as:
+    - Possibility of replay attacks on the SAML payload giving an attacker a secret web session
+    - Increased risk to man-in-the-middle based session hijacking and impersonation attacks
```

Comment:
```suggestion
    - Increased risk of session hijacking and impersonation attacks based on intercepting SAML communications
```
We would need to add another hyphen before "based", which looks awkward to me—suggested this rephrasing instead.